hack/verify-structured-logging.sh

```diff
@@ -52,7 +52,8 @@ done < <(cat "${migrated_packages_file}")
 # TODO: Improve concurrancy here
 ret=0
 for package in "${migrated_packages[@]}"; do
-  logcheck "$KUBE_ROOT/$package" || ret=$?
+    GOOS=linux  logcheck "$KUBE_ROOT/$package" || ret=$?
```

Comment:
little nit: these lines both have more than one space between the env and the command, but the commands are not aligned, instead of dropping the extra space from both lines, let's align the actual command:
```sh
    GOOS=linux   logcheck "$KUBE_ROOT/$package" || ret=$?
    GOOS=windows logcheck "$KUBE_ROOT/$package" || ret=$?
```